src/routes/market/index.svelte

```diff
@@ -99,7 +99,7 @@
       $artworks = [
         ...$artworks,
         ...result.data.artworks.filter(
-          (a) => !$artworks.find((b) => a.id === b.id)
+          (a) => !$artworks.find((b) => a.id === b.id) && !a.is_locked
```

Comment:
This hides them on the market page on the frontend but people could still view the artwork if they know the URL id or slug. Maybe we can set some row permissions in hasura to prevent it from being selected at all.